redhat-developer/developers.redhat.com
# [SURGE 3038] Product Download Hero

### Github Issue Link
* https://github.com/redhat-developer/developers.redhat.com/issues/3038

### Verification Process
1. Create or view existing product download hero
2. Try all content variations
3. Confirm that it matches FE markup here: https://redhat-developer.github.io/rhd-frontend/components/product-download-hero/
